README.md

```diff
@@ -16,180 +18,72 @@ Streisand
+    * [simple-obfs](https://github.com/shadowsocks/simple-obfs) add-on available.
+* A private [Tor](https://www.torproject.org/) bridge relay
+    * [Obfsproxy](https://www.torproject.org/projects/obfsproxy.html.en) with obfs4 available as an add-on.
+* [WireGuard](https://www.wireguard.com/), a modern high-performance protocol.
 
-The server must be accessible using the `$HOME/.ssh/id_rsa` SSH Key, and **root** is used as the connecting user by default. If your provider requires you to SSH with a different user than root (e.g. `ubuntu`) specify the `ANSIBLE_SSH_USER` environmental variable (e.g. `ANSIBLE_SSH_USER=ubuntu`) when you run `./streisand`.
+See also:
+* A [more technical list of features](Features.md) 
+* A [more technical list of services](Services.md)
 
-**Note:** Running Streisand against an existing server can be a destructive action! You will be potentially overwriting configuration files and must be certain that you are affecting the correct machine.
+<a id="cloud-providers"></a>
+## Cloud providers
+* Amazon Web Services (AWS)
+* Microsoft Azure
+* Digital Ocean
+* Google Compute Engine (GCE)
+* Linode
+* Rackspace
 
-### Noninteractive Deployment (Advanced) ###
 
-Alternative scripts and configuration file examples are provided for
-noninteractive deployment, in which all of the required information is passed
-on the command line or in a configuration file.
+#### Other providers
+If you are an expert and can set up a *fresh Ubuntu 16.04 server* elsewhere, Streisand includes code for:
 
-Example configuration files are found under `global_vars/noninteractive`. Copy
-and edit the desired parameters, such as providing API tokens and other choices,
-and then run the appropriate script.
+* Using `ssh` to connect to your remote server from your local machine
+* After connecting to your remote server, you can run Streisand *in place*, called "localhost deployment". This converts the server to a Streisand node.
 
-To deploy a new Streisand server:
+"Localhost deployment" often works better than `ssh` deployment. 
 
-      deploy/streisand-new-cloud-server.sh \
-        --provider digitalocean \
-        --site-config global_vars/noninteractive/digitalocean-site.yml
+For more information, see [the advanced installation instructions](Advanced%20installation.md).
 
-To run the Streisand provisioning on the local machine:
+## Installation
 
-      deploy/streisand-local.sh \
-        --site-config global_vars/noninteractive/local-site.yml
+You need command-line access to a Unix system. You can use Linux, BSD, or macOS; on Windows 10, the Windows Subsystem for Linux (WSL) counts as Linux. 
 
-To run the Streisand provisioning against an existing server:
+Once you're ready, see the [full installation instructions](Installation.md).
 
-      deploy/streisand-existing-cloud-server.sh \
-        --ip-address 10.10.10.10 \
-        --ssh-user root \
-        --site-config global_vars/noninteractive/digitalocean-site.yml
 
 Upcoming Features
```

Comment:
I would vote to remove this whole section. At best its optimistic and at worst its inaccurate :laughing: Either way it probably doesn't belong in the top level README as opposed to somewhere else.

Reply:
I changed to "Things we want to do better"; I'd like to present ourselves as imperfect and looking for help.